test/lsp-completion-test.el

```diff
@@ -79,5 +80,16 @@
              '("F" "daFo" "safo")
              '("F" "daFo" "safo"))))
 
+(ert-deftest lsp-completion-test-get-context ()
+  (cl-labels ((do-test (command)
+                       (let ((last-command command))
+                         (hash-table-values (lsp-completion--get-context '("_"))))))
```

Comment:
you should use something else other than `hash-table` directly here. When we switch to using `plist`, this will become plist.
@yyoncho Do you have any suggestion on which function should we use?

Reply:
> @yyoncho Do you have any suggestion on which function should we use?

I guess one of `lsp:completion-context-trigger-kind` `lsp:completion-context-trigger-character?` should do?